tests/pyccel/scripts/expressions.py

```diff
@@ -0,0 +1,68 @@
+a1 = 2/2
+a2 = 2/2j
+
+a3 = 2//2
+a4 = 2.//2
+a5 = 2//4.
+
+a6 = 2%4
+a7 = 2%4.
+a8 = 2.%4
+
```

Comment:
We should add more tests using the same operator multiple times to ensure the operator treatment order is correct

Reply:
done

Reply:
I meant tests like the one you have added as e2, but for each operator

Reply:
done

Reply:
To make it easier to add more tests, we should only print the results of the various expressions, without making any comparison to a known value. In other words, I would change `print(a1==1.0)` to `print(a1)` and so on. We may need to collect the expressions based on the output type, but this should not be a problem.

Reply:
I have tried that but they were some bugs in how we parse the results in test_pyccel
so i preferred to make the result as a boolean this works as an assert

Reply:
@EmilyBourne I thought we fixed the issue with the output comparison in `test_pyccel` a while ago. Do you remember?

Reply:
yes it still gives me errors in this script

Reply:
I thought so too. I have just had a look. It is pretty basic so I would imagine that the problem is coming from complexes.
A complex in fortran is printed as 2 doubles but in python a completely imaginary complex will be printed as 1 double. This will then give problems for all subsequent results

Reply:
It is therefore probably this line:
`print(a10==10j)`
that is causing problems.

The temporary workaround while waiting for a pull request with a fix would be to define the output type of this object as "float" but it's not a great solution

Reply:
yes i remember the problem is there

Reply:
OK then, let's keep things as they are. We are going to fix it later.

Reply:
As things are meaning without the additional tests? Or just with the boolean results?

Reply:
yes we use the boolean value for the moment and fix the the problem of complex values later

Reply:
So then my initial comment that we need additional tests which have the same operator twice in the expression to ensure that the left to right or right to left precedence is respected still holds

Reply:
@EmilyBourne You're right, we should add more tests before closing this PR. Something like
```
117. / 39. / 3. == 1.   # not 9.
117 // 39 // 3  == 1    # not 9
23  %  12 %  7  == 4    # not 3
2 ** 3 ** 2     == 512  # not 64
```

Reply:
https://github.com/pyccel/pyccel/blob/374d6c5e5b2559c7cab30f88b1a2f8f446004417/tests/pyccel/scripts/expressions.py#L33
I have already added them

Reply:
Ah sorry I missed that. ** seems to be missing though

Reply:
@saidctb Can you add a test with multiple successive power operators `**`?

Reply:
yes I have just added them

Reply:
Perfect, thanks!